docs/source/guides/index.mdx

```diff
@@ -144,7 +144,14 @@ Use a [contract variant](/graphos/platform/schema-management/delivery/contracts/
 TODO
 \*/}
 
-## Deploying a container
+## Deploying the MCP server
+
+There are two ways to deploy and operate the MCP server. 
+
+1. Using the [MCP server container](#deploying-the-mcp-server-container) or binary, which connects to an existing GraphQL API endpoint
+1. Using the [Apollo Runtime container](#deploying-mcp-using-the-apollo-runtime-container), which includes both an MCP server as well as the Apollo router
```

Comment:
```suggestion
1. Using the [Apollo Runtime container](#deploying-mcp-using-the-apollo-runtime-container), which includes both an MCP server as well as the GraphOS Router.
```

Reply:
@lizhennessy - GraphOS is our commercial router. This container is intended to support both OSS, Free, and Commercial customers.

Reply:
@lizhennessy - can you do a final review here? I think we are ready.